one_fm/operations/doctype/mom/mom.js

```diff
@@ -14,25 +14,43 @@ frappe.ui.form.on('MOM', {
 			frm.clear_table("attendees");
 		}
 		if(frm.doc.project){
+			get_project_type(frm, "Project", frm.doc.project);
 			get_poc_list(frm, "Project", frm.doc.project);
 		}
 		frm.refresh_fields("attendees");
+
 	},
 	review_last_mom: function(frm) {
 		if(frm.doc.review_last_mom == 1){
 			frappe.call({
-				method: 'one_fm.operations.doctype.mom.mom.review_last_mom',
+				method: 'frappe.client.get',
```

Comment:
Instesd `frappe.client.get` we can use `frappe.client.get_value` or `frappe.db.get_value('Project', frm.doc.project, 'project_type', function(r){ r.project_type })` or we can set the project type in MOM by adding the field in MOM and set fetch from project.